packages/hooks/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@apollo/react-hooks",
+  "description": "React Apollo Hooks.",
+  "version": "0.1.0-beta.7",
+  "author": "opensource@apollographql.com",
+  "keywords": [
+    "apollo",
+    "graphql",
+    "react",
+    "hooks"
+  ],
+  "license": "MIT",
+  "main": "./lib/react-hooks.cjs.js",
+  "module": "./lib/react-hooks.esm.js",
```

Comment:
Since the `@apollo/react-hooks` package is where we now define `getDataFromTree`, I think we need to apply my commit 58c9f6e6b9c77f26795e47a485252fd875eb0bdb to this `package.json` file, so that React Native will not be confused by the `react-dom/server` dependency.

Reply:
Oh right! I'll get that in - thanks!